cve_bin_tool/cli.py

```diff
@@ -186,8 +186,14 @@ def main(argv=None):
         "-f",
         "--format",
         action="store",
-        choices=["csv", "json", "console", "html", "pdf"],
-        help="update output format (default: console)",
+        help=textwrap.dedent(
+            """\
+            update output format (default: console)
+            specify multiple output formats by using comma (',') as a separator
+            note: don't use spaces between comma (',') and the output formats.
```

Comment:
It's better if we use caps at the start of a sentence.

Reply:
last line? should i capitalize the "N" in note? or the "D" in don't? or both?

Reply:
I initially meant the whole paragraph, but it will be inconsistent with other help texts so let's keep it this way.